api.js

```diff
@@ -171,9 +171,9 @@ class Api extends EventEmitter {
 		const execArgv = this.options.testOnlyExecArgv || process.execArgv;
 		let debugArgIndex = -1;
 
-		// --debug-brk is used in addition to --inspect to break on first line and wait
+		// --inspect-brk is used in addition to --inspect to break on first line and wait
 		execArgv.some((arg, index) => {
-			const isDebugArg = arg === '--inspect' || arg.indexOf('--inspect=') === 0;
+			const isDebugArg = /^--inspect(-brk)?($|=)/.test(arg);
```

Comment:
It's there in Node.js 8: https://nodejs.org/api/debugger.html#debugger_v8_inspector_integration_for_node_js